Packs/GoogleChatViaWebhook/Integrations/GoogleChatViaWebhook/GoogleChatViaWebhook.py

```diff
@@ -0,0 +1,251 @@
+            }]
+        }
+
+        params = {
+            'key': self.key,
+            'token': self.token
+        }
+
+        if threadName:
+            json_data.update({'thread': {'name': threadName}})
+            params.update({'messageReplyOption': 'REPLY_MESSAGE_FALLBACK_TO_NEW_THREAD'})
+
+        res = self._http_request(
+            method='POST',
+            json_data=json_data,
+            raise_on_status=True,
+            url_suffix=f'/messages',
+            params=params
+        )
+        demisto.info(f'Message sent. Response: {res}')
+        return res
+
+
+def test_module(client):
+    """
+    Test command, will send a notification with a static message.
+
+    Args:
+        client (Client): Google Chat client to use.
+
+    Returns:
+        str: 'ok' if test passed, anything else will raise an exception and will fail the test.
+    """
+    try:
+        message = 'Successful test message from Cortex XSOAR'
+        client.send_google_chat_message(message=message, threadName=None)
+        return 'ok'
+    except DemistoException as e:
+        return f'Error: {e}'
+
+
+def send_google_chat_message_command(client: Client, message: str, threadName: Optional[str]) -> CommandResults:
+    """
+    send_google_chat_message command: Sends the Google Chat Message to the provided webhook.
+
+    Args:
+        client (Client): Google Chat client to use.
+        message (str): The message to send to the Google Chat Space.
+
+    Returns:
+        CommandResults/dict: A ``CommandResults`` compatible to return ``return_results()``,
+        which contains the readable_output indicating the message was sent.
+    """
+    try:
+        res = client.send_google_chat_message(message=message, threadName=threadName)
+
+        if res:
+            result = {
+                'Message': res.get('text'),
+                'SpaceName': res.get('space').get('name'),
+                'SpaceDisplayName': res.get('space').get('displayName'),
+                'SpaceType': res.get('space').get('type'),
+                'CreatedTime': res.get('createTime'),
+                'ThreadReply': res.get('threadReply', False),
+                'ThreadName': res.get('thread').get('name'),
+                'Name': res.get('name'),
+                'SenderDisplayName': res.get('sender').get('displayName'),
+                'SenderName': res.get('sender').get('name'),
+                'SenderType': res.get('sender').get('type')
+            }
+
+            res_thread_name = res.get('thread').get('name')
+            markdown = '### Google Chat\n'
+            markdown += tableToMarkdown('Message Webhook', result)
+            results = CommandResults(
+                readable_output=markdown,
+                outputs_prefix='GoogleChatWebhook.Message',
+                outputs_key_field='name',
+                outputs=result
+            )
+
+            return results
+    except DemistoException as e:
+        return f'Error: {e}'
```

Comment:
```suggestion
    res = client.send_google_chat_message(message=message, threadName=threadName)

    if res:
        result = {
            'Message': res.get('text'),
            'SpaceName': res.get('space').get('name'),
            'SpaceDisplayName': res.get('space').get('displayName'),
            'SpaceType': res.get('space').get('type'),
            'CreatedTime': res.get('createTime'),
            'ThreadReply': res.get('threadReply', False),
            'ThreadName': res.get('thread').get('name'),
            'Name': res.get('name'),
            'SenderDisplayName': res.get('sender').get('displayName'),
            'SenderName': res.get('sender').get('name'),
            'SenderType': res.get('sender').get('type')
        }

        res_thread_name = res.get('thread').get('name')
        markdown = '### Google Chat\n'
        markdown += tableToMarkdown('Message Webhook', result)
        results = CommandResults(
            readable_output=markdown,
            outputs_prefix='GoogleChatWebhook.Message',
            outputs_key_field='name',
            outputs=result
        )

        return results
```
No need to add a try/except block, since if an error occurs, the try/except block in the main function will catch it.